### DISCUSSION TITLE
Updated manual

### DISCUSSION THREAD

**Discussion by sumpurnis** [Garux/netradiant-custom#189] [Ideas]
Hi again,

I was wondering if it would be possible to get a bit more pronounced manual on how to use some (if not all) of the features mentioned in netradiant homepage? I see the pipe bending feature, but I can't figure out how it works...

The existing manual seem to be somewhat broken - missing a lot of features and pages. That would be helpful for those who recently started mapping and/or are not yet familiar with features you offer with netradiant.

Thanks in advance!

**Comment by Garux** [comment]
What is 'existing manual' you refer to?
I don't have time and power to devise NRC specific manual from scratch (however this would be sweet).
There is this fresh manual written with NRC in mind, but not going too deeply https://moviebattles.fandom.com/wiki/Mapping_Basics

What's unclear with patch bending? It's not feature on its own, it's showcase of transform tool use.
Transform origin adjustment is simple drag you can observe, vertices selection and rotation snapping are binds documented in menu.help.mouse shortcuts

**Comment by Vioxini00** [comment]
I understand this message. He perhaps refers to this: https://garux.github.io/NRC/ so he refers to explaining textually buttons pressed. I am also a beginner with some previous experience in GTKRadiant and it's hard to guess a lot of things, most go by trial and error. I expect none to do a manual however, but rather by the old method of asking. It would be nice to have a Discord for netradiant custom users so we can help each other with questions.

**Reply by Garux** [reply]
Howdy, I'm in most of idtech mapping discord servers, user id 246624639655215114, feel free to ask, there are many helpful people.
